tests/h/routes_test.py

```diff
@@ -79,6 +79,7 @@ def test_includeme():
              traverse='/{id}'),
         call('api.profile', '/api/profile'),
         call('api.debug_token', '/api/debug-token'),
+        call(u'api.group_member', u'/api/groups/{pubid}/members/me', factory=u'h.models.group:GroupFactory', traverse=u'/{pubid}'),
```

Comment:
There's no need for the `u''` as we're importing `unicode_literals` from `__future__` at the top of the file, also the other tests aren't using it.